packages/fxa-content-server/tests/intern.js

```diff
@@ -76,7 +76,15 @@ const config = {
   fxaPaymentsRoot,
 
   pageLoadTimeout: 20000,
-  reporters: 'runner',
+  reporters: [
```

Comment:
Seemed more useful to make this the default since it would be used in other test suites.